File: python/1134.py
alcoholCount = 0
gasCount = 0
dieselCount = 0

while True:
    x = int(input())
    
    if x == 1:
        alcoholCount += 1
    if x == 2:
        gasCount += 1
    if x == 3:
        dieselCount += 1
    if x == 4:
        break

print('MUITO OBRIGADO')
print(f'Alcool: {alcoholCount}')
print(f'Gasolina: {gasCount}')
print(f'Diesel: {dieselCount}')

File: lua/1134.lua
local counts = {
    ['Alcool'] = 0,
    ['Gasolina'] = 0,
    ['Diesel'] = 0,

}

local codes = {
    [1] = 'Alcool',
    [2] = 'Gasolina',
    [3] = 'Diesel'

}
while true do
    local cod = tonumber(io.read())
    if cod == 4 then
        break
    end
    if codes[cod] then
        local key = codes[cod]
        local value = counts[key]
        counts[key] = value + 1
    end
end

print('MUITO OBRIGADO')

for ind, key  in ipairs(codes) do
    print(string.format('%s: %i',key, counts[key]))
end